{"cvrfdoc":{"document_title":"Red Hat Security Advisory: python security update","document_distribution":"Copyright Â© 2021 Red Hat, Inc. All rights reserved.","document_references":{"reference":[{"description":"https://access.redhat.com/errata/RHSA-2007:1076","type":"Self","url":"https://access.redhat.com/errata/RHSA-2007:1076"},{"description":"http://www.redhat.com/security/updates/classification/#moderate","type":"External","url":"http://www.redhat.com/security/updates/classification/#moderate"}]},"aggregate_severity":"Moderate","document_tracking":{"initial_release_date":"2007-12-10T18:20:00Z","identification":{"id":"RHSA-2007:1076"},"revision_history":{"revision":{"date":"2007-12-10T13:41:00Z","number":"2","description":"Current version"}},"generator":{"date":"2021-08-23T04:29:00Z","engine":"Red Hat SDEngine 3.0.0"},"current_release_date":"2007-12-10T13:41:00Z","version":"2","status":"Final"},"product_tree":{"relationship":[{"relates_to_product_reference":"3AS","product_reference":"python-0:2.2.3-6.8","full_product_name":{"product_id":"3AS:python-0:2.2.3-6.8","cpe":"cpe:/o:redhat:enterprise_linux:3::as","product_name":"python-0:2.2.3-6.8 as a component of Red Hat Enterprise Linux AS version 3"},"relation_type":"Default Component Of"},{"relates_to_product_reference":"3Desktop","product_reference":"python-0:2.2.3-6.8","full_product_name":{"product_id":"3Desktop:python-0:2.2.3-6.8","cpe":"cpe:/o:redhat:enterprise_linux:3::desktop","product_name":"python-0:2.2.3-6.8 as a component of Red Hat Desktop version 3"},"relation_type":"Default Component Of"},{"relates_to_product_reference":"3ES","product_reference":"python-0:2.2.3-6.8","full_product_name":{"product_id":"3ES:python-0:2.2.3-6.8","cpe":"cpe:/o:redhat:enterprise_linux:3::es","product_name":"python-0:2.2.3-6.8 as a component of Red Hat Enterprise Linux ES version 3"},"relation_type":"Default Component Of"},{"relates_to_product_reference":"3WS","product_reference":"python-0:2.2.3-6.8","full_product_name":{"product_id":"3WS:python-0:2.2.3-6.8","cpe":"cpe:/o:redhat:enterprise_linux:3::ws","product_name":"python-0:2.2.3-6.8 as a component of Red Hat Enterprise Linux WS version 3"},"relation_type":"Default Component Of"},{"relates_to_product_reference":"4AS","product_reference":"python-0:2.3.4-14.4.el4_6.1","full_product_name":{"product_id":"4AS:python-0:2.3.4-14.4.el4_6.1","cpe":"cpe:/o:redhat:enterprise_linux:4::as","product_name":"python-0:2.3.4-14.4.el4_6.1 as a component of Red Hat Enterprise Linux AS version 4"},"relation_type":"Default Component Of"},{"relates_to_product_reference":"4Desktop","product_reference":"python-0:2.3.4-14.4.el4_6.1","full_product_name":{"product_id":"4Desktop:python-0:2.3.4-14.4.el4_6.1","cpe":"cpe:/o:redhat:enterprise_linux:4::desktop","product_name":"python-0:2.3.4-14.4.el4_6.1 as a component of Red Hat Enterprise Linux Desktop version 4"},"relation_type":"Default Component Of"},{"relates_to_product_reference":"4ES","product_reference":"python-0:2.3.4-14.4.el4_6.1","full_product_name":{"product_id":"4ES:python-0:2.3.4-14.4.el4_6.1","cpe":"cpe:/o:redhat:enterprise_linux:4::es","product_name":"python-0:2.3.4-14.4.el4_6.1 as a component of Red Hat Enterprise Linux ES version 4"},"relation_type":"Default Component Of"},{"relates_to_product_reference":"4WS","product_reference":"python-0:2.3.4-14.4.el4_6.1","full_product_name":{"product_id":"4WS:python-0:2.3.4-14.4.el4_6.1","cpe":"cpe:/o:redhat:enterprise_linux:4::ws","product_name":"python-0:2.3.4-14.4.el4_6.1 as a component of Red Hat Enterprise Linux WS version 4"},"relation_type":"Default Component Of"}],"branch":[{"name":"Red Hat Enterprise Linux","type":"Product Family","branch":[{"name":"Red Hat Enterprise Linux AS version 3","full_product_name":{"product_id":"3AS","cpe":"cpe:/o:redhat:enterprise_linux:3::as","product_name":"Red Hat Enterprise Linux AS version 3"},"type":"Product Name"},{"name":"Red Hat Desktop version 3","full_product_name":{"product_id":"3Desktop","cpe":"cpe:/o:redhat:enterprise_linux:3::desktop","product_name":"Red Hat Desktop version 3"},"type":"Product Name"},{"name":"Red Hat Enterprise Linux ES version 3","full_product_name":{"product_id":"3ES","cpe":"cpe:/o:redhat:enterprise_linux:3::es","product_name":"Red Hat Enterprise Linux ES version 3"},"type":"Product Name"},{"name":"Red Hat Enterprise Linux WS version 3","full_product_name":{"product_id":"3WS","cpe":"cpe:/o:redhat:enterprise_linux:3::ws","product_name":"Red Hat Enterprise Linux WS version 3"},"type":"Product Name"},{"name":"Red Hat Enterprise Linux AS version 4","full_product_name":{"product_id":"4AS","cpe":"cpe:/o:redhat:enterprise_linux:4::as","product_name":"Red Hat Enterprise Linux AS version 4"},"type":"Product Name"},{"name":"Red Hat Enterprise Linux Desktop version 4","full_product_name":{"product_id":"4Desktop","cpe":"cpe:/o:redhat:enterprise_linux:4::desktop","product_name":"Red Hat Enterprise Linux Desktop version 4"},"type":"Product Name"},{"name":"Red Hat Enterprise Linux ES version 4","full_product_name":{"product_id":"4ES","cpe":"cpe:/o:redhat:enterprise_linux:4::es","product_name":"Red Hat Enterprise Linux ES version 4"},"type":"Product Name"},{"name":"Red Hat Enterprise Linux WS version 4","full_product_name":{"product_id":"4WS","cpe":"cpe:/o:redhat:enterprise_linux:4::ws","product_name":"Red Hat Enterprise Linux WS version 4"},"type":"Product Name"}]},{"name":"python-0:2.2.3-6.8","full_product_name":{"product_id":"python-0:2.2.3-6.8","cpe":"cpe:/o:redhat:enterprise_linux:3::as","product_name":"python-2.2.3-6.8.src.rpm"},"type":"Product Version"},{"name":"python-0:2.3.4-14.4.el4_6.1","full_product_name":{"product_id":"python-0:2.3.4-14.4.el4_6.1","cpe":"cpe:/o:redhat:enterprise_linux:4::as","product_name":"python-2.3.4-14.4.el4_6.1.src.rpm"},"type":"Product Version"}]},"document_publisher":{"issuing_authority":"Red Hat Product Security","contact_details":"secalert@redhat.com","type":"Vendor"},"vulnerability":[{"notes":{"note":"pcre integer overflow"},"cve":"CVE-2006-7228","references":{"reference":[{"description":"CVE-2006-7228","url":"https://access.redhat.com/security/cve/CVE-2006-7228"},{"description":"bz#383371: CVE-2006-7228 pcre integer overflow","url":"https://bugzilla.redhat.com/show_bug.cgi?id=383371"}]},"release_date":"2007-11-07T00:00:00Z","involvements":{"involvement":{"party":"Vendor","status":"Completed"}},"product_statuses":{"status":{"product_id":["3AS:python-0:2.2.3-6.8","3Desktop:python-0:2.2.3-6.8","3ES:python-0:2.2.3-6.8","3WS:python-0:2.2.3-6.8","4AS:python-0:2.3.4-14.4.el4_6.1","4Desktop:python-0:2.3.4-14.4.el4_6.1","4ES:python-0:2.3.4-14.4.el4_6.1","4WS:python-0:2.3.4-14.4.el4_6.1"],"type":"Fixed"}},"remediations":{"remediation":{"description":"Before applying this update, make sure that all previously-released\nerrata relevant to your system have been applied.  \nThis update is available via Red Hat Network.  Details on how to use \nthe Red Hat Network to apply this update are available at\nhttp://kbase.redhat.com/faq/FAQ_58_10188","type":"Vendor Fix","url":"https://access.redhat.com/errata/RHSA-2007:1076"}},"threats":{"threat":{"description":"Important","type":"Impact"}},"discovery_date":"2007-11-08T00:00:00Z","ordinal":"1"},{"notes":{"note":"python off-by-one locale.strxfrm() (possible memory disclosure)"},"cve":"CVE-2007-2052","references":{"reference":[{"description":"CVE-2007-2052","url":"https://access.redhat.com/security/cve/CVE-2007-2052"},{"description":"bz#235093: CVE-2007-2052 python off-by-one locale.strxfrm() (possible memory disclosure)","url":"https://bugzilla.redhat.com/show_bug.cgi?id=235093"}]},"release_date":"2007-04-02T00:00:00Z","involvements":{"involvement":{"party":"Vendor","status":"Completed"}},"product_statuses":{"status":{"product_id":["3AS:python-0:2.2.3-6.8","3Desktop:python-0:2.2.3-6.8","3ES:python-0:2.2.3-6.8","3WS:python-0:2.2.3-6.8","4AS:python-0:2.3.4-14.4.el4_6.1","4Desktop:python-0:2.3.4-14.4.el4_6.1","4ES:python-0:2.3.4-14.4.el4_6.1","4WS:python-0:2.3.4-14.4.el4_6.1"],"type":"Fixed"}},"remediations":{"remediation":{"description":"Before applying this update, make sure that all previously-released\nerrata relevant to your system have been applied.  \nThis update is available via Red Hat Network.  Details on how to use \nthe Red Hat Network to apply this update are available at\nhttp://kbase.redhat.com/faq/FAQ_58_10188","type":"Vendor Fix","url":"https://access.redhat.com/errata/RHSA-2007:1076"}},"threats":{"threat":{"description":"Low","type":"Impact"}},"discovery_date":"2007-04-02T00:00:00Z","ordinal":"2"},{"notes":{"note":"python imageop module heap corruption"},"cve":"CVE-2007-4965","references":{"reference":[{"description":"CVE-2007-4965","url":"https://access.redhat.com/security/cve/CVE-2007-4965"},{"description":"bz#295971: CVE-2007-4965 python imageop module heap corruption","url":"https://bugzilla.redhat.com/show_bug.cgi?id=295971"}]},"release_date":"2007-09-16T00:00:00Z","involvements":{"involvement":{"party":"Vendor","status":"Completed"}},"product_statuses":{"status":{"product_id":["3AS:python-0:2.2.3-6.8","3Desktop:python-0:2.2.3-6.8","3ES:python-0:2.2.3-6.8","3WS:python-0:2.2.3-6.8","4AS:python-0:2.3.4-14.4.el4_6.1","4Desktop:python-0:2.3.4-14.4.el4_6.1","4ES:python-0:2.3.4-14.4.el4_6.1","4WS:python-0:2.3.4-14.4.el4_6.1"],"type":"Fixed"}},"remediations":{"remediation":{"description":"Before applying this update, make sure that all previously-released\nerrata relevant to your system have been applied.  \nThis update is available via Red Hat Network.  Details on how to use \nthe Red Hat Network to apply this update are available at\nhttp://kbase.redhat.com/faq/FAQ_58_10188","type":"Vendor Fix","url":"https://access.redhat.com/errata/RHSA-2007:1076"}},"threats":{"threat":{"description":"Moderate","type":"Impact"}},"discovery_date":"2007-09-18T00:00:00Z","ordinal":"3"}],"document_notes":{"note":["Updated python packages that fix several security issues are now available\nfor Red Hat Enterprise Linux 3 and 4.\nThis update has been rated as having moderate security impact by the Red\nHat Security Response Team.","Python is an interpreted, interactive, object-oriented programming\nlanguage.\nAn integer overflow flaw was discovered in the way Python's pcre module\nhandled certain regular expressions. If a Python application used the pcre\nmodule to compile and execute untrusted regular expressions, it may be\npossible to cause the application to crash, or allow arbitrary code\nexecution with the privileges of the Python interpreter. (CVE-2006-7228)\nA flaw was discovered in the strxfrm() function of Python's locale module.\nStrings generated by this function were not properly NULL-terminated. This\nmay possibly cause disclosure of data stored in the memory of a Python\napplication using this function. (CVE-2007-2052)\nMultiple integer overflow flaws were discovered in Python's imageop module.\nIf an application written in Python used the imageop module to process\nuntrusted images, it could cause the application to crash, enter an\ninfinite loop, or possibly execute arbitrary code with the privileges of\nthe Python interpreter. (CVE-2007-4965)\nUsers of Python are advised to upgrade to these updated packages, which\ncontain backported patches to resolve these issues.","This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original."]},"document_type":"Security Advisory"}}